frontend/src/app/max-validator.directive.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable:deprecation (false positive) */
```

Comment:
Is this related?

Reply:
No. I must have missed it the last time I made an upgrade.